sdk/core/core-lro/package.json

```diff
@@ -65,15 +65,19 @@
   },
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build": "npm run build:es6 && npm run build:nodebrowser && npm run build:samples && npm run extract-api",
+    "build": "npm run build:ts && npm run build:nodebrowser && api-extractor run --local",
     "build:samples": "tsc -p tsconfig.samples.json",
-    "build:es6": "tsc -p tsconfig.json",
+    "build:ts": "tsc -p .",
     "build:nodebrowser": "rollup -c 2>&1",
-    "build:test": "npm run build:es6 && rollup -c rollup.test.config.js 2>&1",
+    "build:node": "npm run build:ts && cross-env ONLY_NODE=true rollup -c 2>&1 && npm run extract-api",
+    "build:browser": "npm run build:ts && cross-env ONLY_BROWSER=true rollup -c 2>&1",
```

Comment:
Looks like these two are not being used in this file.

Reply:
What do you mean? These command are for local or in ci if we only want to run node/browser tests

Reply:
I don't think we intend any other script in the file to call these commands